src/Outsystems.SetupTools/Functions/Install-OSServer.ps1

```diff
@@ -166,8 +175,14 @@ function Install-OSServer
         }
         else
         {
-            $installPlatformServer = $false
-            LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 0 -Message "OutSystems platform server already installed with the specified version $osVersion"
+            if ($Force.IsPresent)
+            {
+                LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 0 -Message "Force switch specified. We will reinstall!!"
+            }else{
+                $installPlatformServer = $false
+                LogMessage -Function $($MyInvocation.Mycommand) -Phase 1 -Stream 0 -Message "OutSystems platform server already installed with the specified version $osVersion"
```

Comment:
Move this message after the "else" on line 178 so we know that OutSystems is already installed with the right version